dwa_local_planner/src/dwa_planner.cpp

```diff
@@ -329,7 +331,7 @@ namespace dwa_local_planner {
       path_align_costs_.setScale(1.0);
```

Comment:
F: we should use `max_backward_sq_dist_` in the check up here too; otherwise,if `MIN_GOAL_DIST_SQ < sq_dist < max_backward_sq_dist_` , the robot won't be able to turn properly in a narrow aisle (-> either it gets stuck, or it moves backwards)